resources/META-INF/plugin.xml

```diff
@@ -21,7 +21,7 @@
 
   <category>Custom Languages</category>
   <version>SNAPSHOT</version>
-  <idea-version since-build="201.7223" until-build="202.8194.7"/>
+  <idea-version since-build="201.7223" until-build="203.*"/>
```

Comment:
Should we be keeping this more specific? I can't remember the details but was .* causing problems for another team?

Reply:
It should be more specific in product-matrix.json but we want the check-in version of plugin.xml to be unbounded for debugging. Good catch, though -- I think I left it unbounded in product-matrix. Will fix later.